docsrc/install.rst

```diff
@@ -47,10 +34,9 @@ Alternatively, you can build a wheel package on your platform from scratch using
 
 .. code-block:: bash
 
-	pip install setuptools wheel # if wheel is not installed
-	python setup.py bdist_wheel
-	pip install dist/jurity-X.X.X-py3-none-any.whl
-
+    pip install setuptools wheel # if wheel is not installed
```

Comment:
If we are removing wheel installation, this should go?

Reply:
You still need this if you're building the wheel yourself from source?